frontend/src/components/page_content/advisor-contact-content.js

```diff
@@ -14,6 +14,10 @@ import {
 } from "../../redux/selectors";
 
 class AdvisorContactContent extends React.Component {
+  componenDidMount() {
```

Comment:
this shouldn't be necessary. the step is already being reset in the constructor in page template component

Reply:
This is to reset the left nav and current page when you change pages in the navbar.  When you're doing navigation on the main nav the set step does not get reset with NavLink.  So this resets the step.  I think NavLink is the way to go since it automatically highlights the active element without having to figure out which nav item to keep active and it may be beneficial that it doesn't reload everything each time you change.

Reply:
The choice to switch to the a tag was so that the components did refresh so that the constructor is called each time to reset the step from the page template component. you can use the css selector :active to see if the link is active